documentation/docs/strategies/avellaneda-market-making.md

```diff
@@ -10,7 +10,7 @@ tags:
 
 ## 📝 Summary
 
-This strategy implements the market making strategy described in the classic paper [High-frequency Trading in a Limit Order Book](https://people.orie.cornell.edu/sfs33/LimitOrderBook.pdf) written by Marco Avellaneda and Sasha Stoikov. It allows users to directly adjust the kappa, gamma, and eta parameters described in the paper. It also features a simplified mode that allows the user to enter min/max spread parameters that continually recalculate the advanced parameters.
+This strategy implements a market making strategy described in the classic paper [High-frequency Trading in a Limit Order Book](https://people.orie.cornell.edu/sfs33/LimitOrderBook.pdf) written by Marco Avellaneda and Sasha Stoikov. It allows users to directly adjust the `gamma` parameter described in the paper. It also features an order book liquidity estimator calculating the `alpha` and `kappa` parameters automatically. Additionally, the strategy implements an order size adjustment algorithm and its `eta` parameter as described in [Optimal High-Frequency Market Making](http://stanford.edu/class/msande448/2018/Final/Reports/gr5.pdf). The strategy is implemented to be used either in fixed timeframes or to be ran indefinitely.
```

Comment:
Use the same naming convention as the table below for the greek parameters, so that users can more easily cross-reference between them:

- `gamma` => `risk factor` (𝛾) "gamma"
- `eta` => `order_amount_shape_factor` (η) "eta"
- `alpha` => ?
- `kappa` => ?